bigquery/google/cloud/bigquery/__init__.py

```diff
@@ -130,6 +131,7 @@
     "Encoding",
     "QueryPriority",
     "SchemaUpdateOption",
+    "StandardSqlDataTypes",
```

Comment:
This reminds me: let's add an Enums section to the API reference docs, too.

https://github.com/googleapis/google-cloud-python/blob/master/bigquery/docs/reference.rst

Reply:
Added. Also had to inject an additional header line to the new enum's docstring, because Sphinx complained about a duplicate object description (took the first line - the INT64 member - which is of course directly copied from the gapic enum).